sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FieldValue.java

```diff
@@ -228,32 +223,17 @@ public SelectionMarkState asSelectionMarkState() {
     }
 
     /**
-     * Gets the value of the field as a Country field.
-     *
-     * @return the value of the field as Country.
-     * @throws UnsupportedOperationException if {@link FieldValue#getValueType()} is not
-     * {@link FieldValueType#COUNTRY}.
-     */
-    public String asCountry() {
-        if (COUNTRY != this.getValueType()) {
-            throw logger.logExceptionAsError((new UnsupportedOperationException(String.format(
-                "Cannot get field as a %s from field value of type %s", COUNTRY, this.getValueType()))));
-        }
-        return this.formFieldCountry;
-    }
-
-    /**
-     * Gets the value of the field as a gender value.
+     * Gets the value of the field as a COUNTRY_REGION field.
      *
-     * @return the value of the field as a gender value.
+     * @return the value of the field as COUNTRY_REGION.
```

Comment:
using COUNTRY_REGION seems strange here, maybe "as a country or region in the world"